catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -96,31 +96,31 @@
         descriptors.add(new AttributeDescriptorImpl(DERIVED_RESOURCE_DOWNLOAD_URL,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
```

Comment:
These derived URLs/URIs should be tokenized or deprecated. Indexing only the full URI/URL is not very useful from a query standpoint.